themes/gatsby-theme-blog/gatsby-node.js

```diff
@@ -39,6 +40,55 @@ exports.onPreBootstrap = ({ store }, themeOptions) => {
   })
 }
 
+const mdxResolverPassthrough = fieldName => async (
+  source,
+  args,
+  context,
+  info
+) => {
+  const type = info.schema.getType(`Mdx`)
+  const mdxNode = context.nodeModel.getNodeById({
+    id: source.parent,
+  })
+  const resolver = type.getFields()[fieldName].resolve
+  const result = await resolver(mdxNode, args, context, {
+    fieldName,
+  })
+  return result
+}
+exports.sourceNodes = ({ actions, schema }) => {
+  const { createTypes } = actions
+  createTypes(
+    schema.buildObjectType({
+      name: `BlogPost`,
+      fields: {
+        id: { type: `ID!` },
+        title: {
+          type: `String!`,
+        },
+        date: { type: `Date`, extensions: { dateformat: {} } },
+        tags: { type: `[String]!` },
+        keywords: { type: `[String]!` },
```

Comment:
Maybe a dumb question but what's the difference between *tags* and *keywords*?

Reply:
keywords seems to be for SEO purposes while I'd expect tags to be user-facing. I'm open to removing this

Reply:
^yep. keywords are tag-oriented, for tags we could do tag filtering later.